client/src/App.css

```diff
@@ -23,6 +23,48 @@
   color: #61dafb;
 }
 
+.text-align-center {
+  text-align: center;
+  top: 50%;
+  line-height: 95px !important;
```

Comment:
Only use common CSS classes for this file.

here, this CSS class should only align text to the center.
other styles added are not relevant to the class name. 

So, if it's custom to the particular component then added it with the component.

Reply:
yes